scripts/secnetperf.ps1

```diff
@@ -235,19 +242,29 @@ if ($isWindows) {
 Configure-DumpCollection $Session
 
 # Install any dependent drivers.
-if ($useXDP) { Install-XDP $Session $RemoteDir }
+if ($useXDP -and $isWindows) { Install-XDP $Session $RemoteDir }
```

Comment:
This just made me realize, we won't be able to differentiate between 'Windows QUIC + XDP' and 'Linux QUIC + XDP' because we name them 'QUIC+XDP' in the data file (I think). @ProjectsByJackHe do we need to make the `$io` names unique?